### ISSUE TITLE
Dyanamically create AutomationID for the Xamarin.Forms controls

### ISSUE THREAD

**Issue by psatheesh18** [xamarin/Xamarin.Forms#5899]
### Steps to Reproduce

1. Setting the automationId dynamically for the same UIelement. 
2. Run the sample file (https://github.com/xamarin/Xamarin.Forms/files/3072440/sample.zip).
3. Once it deployed to the device or emulator press the button it will show the error i have mentioned in the screenshot

### Expected Behavior
Dynamically AutomationId to be changed
### Actual Behavior
AutomationId can be set only once. I have tried with setting up the AutomationProperties.SetName but it was not working ,So i have tried with device specified using switch case .So i need to use automationid for android device to setup accessibility .
### Basic Information
I need to change the automationid randomly to provide accessibility in the android .
- Version with issue:  Xamarin.Forms 2.3.3 and above
- Last known good version: 
- IDE: Visual Studio 15.9.9
- Platform Target Frameworks: 
  - iOS: Nil 
  - Android: All versions 
  - UWP: Nil  
- Android Support Library Version: 
- Nuget Packages:Xamarin.Forms 2.3 and above
- Affected Devices: All Android devices

### Screenshots
![image](https://user-images.githubusercontent.com/36506007/56019516-77b2f980-5d22-11e9-961d-1f5315779c72.png)

### Reproduction Link

[sample.zip](https://github.com/xamarin/Xamarin.Forms/files/3072440/sample.zip)

**Comment by samhouts** [comment]
@psatheesh18 The error that you're reporting is by design. In what context does `AutomationProperties.SetName` or `AutomationProperties.SetHint` not work?

**Comment by psatheesh18** [comment]
I have already attached the sample project as sample.zip in the reproduction link .

**Comment by samhouts** [comment]
@psatheesh18 We are not prepared to change the one-time binding for `AutomationId`. Please open a new issue describing why `AutomationProperties.SetName` or `AutomationProperties.SetHint` are not working for you. Thanks!
